src/SprykerEco/Zed/UnzerGui/Communication/Form/UnzerCredentialsEditForm.php

```diff
@@ -15,11 +15,14 @@ class UnzerCredentialsEditForm extends UnzerCredentialsCreateForm
 {
     /**
      * @param \Symfony\Component\Form\FormBuilderInterface $builder
-     * @param array<string, string> $choices
+     * @param array<string, int> $choices
      *
-     * @return \SprykerEco\Zed\UnzerGui\Communication\Form\UnzerCredentialsCreateForm
+     * @return $this
      */
-    protected function addTypeField(FormBuilderInterface $builder, array $choices = []): UnzerCredentialsCreateForm
+    protected function addTypeField(
+        FormBuilderInterface $builder,
+        array $choices = []
+    ): UnzerCredentialsEditForm
```

Comment:
```suggestion
    protected function addTypeField(FormBuilderInterface $builder, array $choices = [])
```

Because of this RFC: https://spryker.atlassian.net/wiki/spaces/RFC/pages/570195995/ACCEPTED+RFC+self+vs+none

Reply:
Fixed.